config/config.yaml

```diff
@@ -0,0 +1 @@
+shkeptnspecversion: "0.2.1"
```

Comment:
i think the current spec version is 0.2.3, let's change it to this, to have the correct one on master after merging

Reply:
sure I will update it

Reply:
but we could also take this opportunity to properly test the new pipeline after merging this PR